package.json

```diff
@@ -35,6 +35,7 @@
   "author": "Denis Akiyakov <newsiberian2015@yandex.ru>",
   "license": "MIT",
   "dependencies": {
+    "@apollo/client": "^3.0.0-beta.41",
```

Comment:
This should probably be a peer dependency?

Reply:
Maybe! So far have just been trying to restore the functionality, then consider NPM hygiene aspects.

Thanks for sharing https://github.com/newsiberian/apollo-link-token-refresh/issues/15 - 
hadn't seen that yet.

Reply:
![image](https://user-images.githubusercontent.com/182515/78792707-c638b980-79b1-11ea-99a3-c18f420331f1.png)

Small update, I tried moving `"@apollo/client": "^3.0.0-beta.43"` from `dependencies` to only `peerDependencies`, but this causes TypeScript to complain at compile time.

If I understand what you're saying correctly, it should be more idiomatic for a library to use a peer dep vs a standard dependency.
Meanwhile I see another popular project, react-router, also uses rollup and uses [peerDeps](https://github.com/ReactTraining/react-router/blob/402ecabdc94e5aeb657c593d8af200625a09cdfe/packages/react-router/package.json#L36). Seems like react-router doesn't use TypeScript though - maybe that's going to be the hurdle on this.

Your insight would appreciated :)